src/device.hpp

```diff
@@ -528,6 +528,9 @@ struct cvk_device : public _cl_device_id,
         return m_max_first_cmd_group_size;
     }
 
+    const std::string get_spirv_arch() const { return m_spirv_arch; }
```

Comment:
```suggestion
```

No longer needed if we introduce `address_bits`.

Reply:
Done